client/src/js/references/components/Import.js

```diff
@@ -117,7 +120,7 @@ class ImportReference extends React.Component {
                         style={{ marginBottom: "10px" }}
                     />
                     <UploadBar onDrop={this.handleDrop} message={message} style={fileErrorStyle} />
-                    <ReferenceForm state={this.state} onChange={this.handleChange} toggle={this.toggleCheck} />
+                    {ReferenceForm(this.state)}
```

Comment:
Create component properly using JSX. Looks at how you did it in `Clone.js` and `Create.js` if unclear.